### PR TITLE
Add new package: py-coapthon3

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
Can you rebase/merge develop to fix the vermin tests?

File: var/spack/repos/builtin/packages/py-coapthon3/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class PyCoapthon3(PythonPackage):
+    """CoAPthon3 is a porting to python3 of my CoAPthon library.
+    CoAPthon3 is a python3 library to the CoAP protocol compliant
+    with the RFC. Branch is available for the Twisted framework."""
+
+    homepage = "https://github.com/Tanganelli/CoAPthon3/"
+    url      = "https://github.com/Tanganelli/CoAPthon3/archive/1.0.1.tar.gz"
+
+    version('1.01',  sha256='b90a410cf35f18c0fb2620baf44a3283a640d40a299c31c7d184a69c1dbec105')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-coapthon3/package.py]: I would remove this version. It's not listed among the releases (only tags), so I assume it was committed by mistake?

**Review by alalazo — CHANGES_REQUESTED** [review]
Can you please double check the version in the comment?

File: var/spack/repos/builtin/packages/py-coapthon3/package.py (hunk 1 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/py-coapthon3/package.py]: Fixed in https://github.com/spack/spack/pull/18990/commits/45d4588dc189ca628f576a0676e4f01c07d9b0db

**Review by alalazo — APPROVED** [review]

**Pull request merged**